### ISSUE TITLE
Suggestion: Change USB-C connector to a widely available and cheaper 24p one 

### ISSUE THREAD

**Issue by Ardakilic** [alvarop/usb_c_cable_tester#20]
Hello,

First of all, thank you for open-sourcing the project.

Today I have built the tester myself, and realized the usb-c connector that I've purchased does not fit properly to the case.

This is because the pinout is somehow different.

After confirming the pins match, I have bought this: $1.91 / 10 pcs:

https://www.aliexpress.com/item/32879436811.html 

![image](https://github.com/alvarop/usb_c_cable_tester/assets/2063957/696a8a30-8185-45c9-a364-f9a610b71138)

![image](https://github.com/alvarop/usb_c_cable_tester/assets/2063957/6ca00fcf-f8bd-48b6-8ace-d4a771913d31)

![image](https://github.com/alvarop/usb_c_cable_tester/assets/2063957/4d9c4c20-1c37-435f-ba9b-0ed6980ac047)

However, today I realized the ground shielding does not fit. 

I used a wrench and my soldering techniques. It fits and works.

![image](https://github.com/alvarop/usb_c_cable_tester/assets/2063957/b03636c5-e476-474c-8fff-2769eb99e96f)

![image](https://github.com/alvarop/usb_c_cable_tester/assets/2063957/9fa9d7ed-fc48-46ff-af8b-24ac0f769ad6)

To change to the proper connector, I searched, and in Aliexpress, there are only 2 sellers, which I believe sells the item and fits to the footprint. I thought these should be widely available.

The closest one, which I could find available is this, and it costs $1.4 per part, https://www.aliexpress.com/item/1005005376413109.html

And it's almost 10 times more expensive than what I've bought, even more expensive than LCSC.

There are various connectors that are 24 pin and pinouts match 1:1, and available at LCSC, where the footprint matches, widely available in other vendors, and they are way cheaper:

For example:

* $0.52 per connector, and the most available part on the category: https://www.lcsc.com/product-detail/USB-Connectors_XUNPU-TYPEC-324-BCP24_C2835314.html
* $0.25 per connector- https://www.lcsc.com/product-detail/USB-Connectors_Lian-Xin-Technology-XUBF-0336-B07_C7527707.html
* $0.46 per connector- https://www.lcsc.com/product-detail/USB-Connectors_SHOU-HAN-TYPE-C-24P-QCHT_C456013.html
* $0.45 per connector- https://www.lcsc.com/product-detail/USB-Connectors_Hanbo-Electronic-MC-263-H20_C6332283.html
* $0.36 per connectorhttps://www.lcsc.com/product-detail/USB-Connectors_HCTL-HC005-1A1H1A103-0HR_C2894904.html
* Discontinued, but you get the idea: https://www.lcsc.com/product-detail/USB-Connectors_XUNPU-TYC-324_C381479.html

There are a lot of more connectors, these are just the results of my humble 5 minutes of searching.

The greater issue is that the lack of availability of the part, apart from the price, as I mentioned.

I reside in Turkey, and my best bet is Aliexpress. After that there is Farnell Turkey, which they don't stock much, and ozdisan.com , and they both lack the connector.

And in Aliexpress, I found only two sellers, as I mentioned.

The pinout is same for both 24 pins, as I checked the footprint: Both SMD 12 pins, and the through hole other 12 pins, so I see no problem on switching the footprint to this widely available footprint model.

I'd appreciate if this is considered.

Thanks in advance!

**Comment by exetico** [comment]
Hi, 

I just stumbled upon the project, and noticed your issue, as I noticed the pretty high price of the C2761408, currently used in the design (8,43 USD for 10 Qty).

So I'll just bump this, so I can follow the process. It's the most expensive part of the BOM, so I guess there's many options to bring down the cost a bit, by replacing the connector with a cheaper one, as you're also suggestion. I'm not going to risk ordering it with another connector, without having a input from the original author, though. 😄 

Thanks for bringing this to the table.

**Comment by Ardakilic** [comment]
@alvarop Thanks for the reply. 

The main issue I find with the current footprint is that it's not a rather generic one, so you're stuck with less brands (both in JLC/LCSC and other vendors). All of the links I've shared (and even more on LCSC's inventory) share the same footprint. That's why I suggested in the first place. 

Of course, the price is a bonus, however, my main concern is the availability, and as you mentioned, current stock does not represent the future stock, that's why I believe a more generic footprint brings more alternatives and availability to the table.
